graf3d/x3d/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(libname X3d)
 
 include_directories(${X11_INCLUDE_DIR})
 
-ROOT_GENERATE_DICTIONARY(G__X3D TViewerX3D.h TX3DFrame.h MODULE ${libname} LINKDEF LinkDef.h DEPENDENCIES Graf3d Gui)
```

Comment:
We probably should be using `G__${libname}` all over the place to keep things consistent.